go.mod

```diff
@@ -163,3 +161,5 @@ replace github.com/sirupsen/logrus => github.com/jtnord/logrus v1.4.2-0.20190423
 replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v21.1.0+incompatible
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
+
+replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
```

Comment:
@ccojocar that seems to make the difference. For some reason, bank-vaults also needs a replace. Really this gets more and more screwed up. All these replace directives are a pita.